src/utils/send-message.ts

```diff
@@ -0,0 +1,33 @@
+import { isProd } from '~/utils/node-env'
+
+// TODO: set correct targetOrigin of the parent window
+const TARGET_ORIGIN = '*'
```

Comment:
Is this planned to be resolved here or in a future PR? 👀

Reply:
It's been that way since the beginning 😆